### ISSUE TITLE
GitHub "View Pull Requests..." > "Fetch to pr/ branch" only works for HTTPS repos

### ISSUE THREAD

**Issue by djc5166** [gitextensions/gitextensions#10867]
### Environment

- Git Extensions 4.0.2.16100
- Build 25100ec1f7c8da8f798613da74826038af81a50e
- Git 2.40.0.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.15
- DPI 96dpi (no scaling)
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

Trying to manage github pull requests through gitextensions. While the initial windows successfully lists all the PRs, all the commands on the "View Pull Requests..." dialog fail to connect to github.com unless the repo is an HTTPS repo. All normal functions are working under a SSH repo.

Tried with multiple repos and PRs.

ssh Repo:
```
"C:\Program Files\Git\bin\git.exe" fetch --no-tags --progress git://github.com/tshirt102/dsfix.git master:pr/n44_master
fatal: unable to connect to github.com:
github.com[0: 140.82.113.3]: errno=Unknown error

Done

Press Enter or Esc to exit...
```

HTTPS Repo:
```
"C:\Program Files\Git\bin\git.exe" fetch --no-tags --progress https://github.com/tshirt102/dsfix.git master:pr/n44_master
remote: Enumerating objects: 3, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 3 (delta 1), reused 1 (delta 1), pack-reused 2
Unpacking objects: 100% (3/3), 985 bytes | 46.00 KiB/s, done.
From https://github.com/tshirt102/dsfix
 * [new branch]      master     -> pr/n44_master
Done

Press Enter or Esc to exit...
```

Manually running the same fetch command with "git@github.com:" works

```
$ git fetch --no-tags --progress git://github.com/tshirt102/dsfix.git master:pr/n44_master
fatal: unable to connect to github.com:
github.com[0: 140.82.113.4]: errno=Unknown error


djc5166@Rukia MINGW64 /i/src/dsfix[PeterTH] (master)
$ git fetch --no-tags --progress git@github.com:tshirt102/dsfix.git master:pr/n44_master
From github.com:tshirt102/dsfix
 * [new branch]      master     -> pr/n44_master
 ```

So I guess the question is where it gitextensions getting git:// from? Or is this a GitHub issue?

### Steps to reproduce

- Add SSH github repo (with active PRs) to gitextensions
- Select GitHub>"View Pull Requests..."
- Select a PR
- Select "Fetch to pr/ branch"
- git fetch command reports it fails to connect to github.com

**Comment by pmiossec** [comment]
>So I guess the question is where it gitextensions getting git:// from? Or is this a GitHub issue?

The `git://` protocol is indeed [no more supported by GitHub since march 2022](https://github.blog/2021-09-01-improving-git-protocol-security-github/) and should not be used anymore by GitExtensions....

**Comment by gerhardol** [comment]
ssh is still supported: git@github.com:gitextensions/gitextensions.git
The git: protocol is not supported.

I only expect though that https:// will ever work with the GitHub plugin, it should probably be disabled for ssh repos.

**Comment by pmiossec** [comment]
@djc5166 Please have a look at my request: https://github.com/gitextensions/gitextensions/pull/10871#issuecomment-1505264898

**Comment by gerhardol** [comment]
Included in release/4.1, to be added to master soon
